spec/services/atmosphere/vm_updater_spec.rb

```diff
@@ -11,10 +11,10 @@
   let(:updated_vm) { Atmosphere::VirtualMachine.find_by(id_at_site: 'id_at_site') }
 
   before do
-    allow(Atmosphere::VirtualMachineTemplate)
-      .to receive(:find_by)
-        .with(compute_site: cs, id_at_site: "vmt_id_at_site")
-          .and_return(vmt)
+    allow(Atmosphere::VirtualMachineTemplate).
+      to receive(:find_by).
+      with(compute_site: cs, id_at_site: "vmt_id_at_site").
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.